types/foundry-vtt-api/types/apps/forms/item.d.ts

```diff
@@ -0,0 +1,49 @@
+declare interface ItemSheetData extends BaseEntitySheetData {
+	item?: Item;
```

Comment:
<!--
  1 failure:  The root .editorc...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: types/foundry-vtt-api/types/apps/forms/item.d.ts;
  Line: 2;
-->
- :no_entry_sign: The root .editorconfig style specifies spaces for whitespace. Please use spaces in new or changed types.